src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -155,30 +182,35 @@ public void run() {
 			boolean successful = true;
 			for (final Node node : (SectionNode) databases) {
 				if (node instanceof SectionNode) {
-					final SectionNode n = (SectionNode) node;
-					final String type = n.getValue("type");
+					SectionNode section = (SectionNode) node;
+					String type = section.getValue("type");
 					if (type == null) {
 						Skript.error("Missing entry 'type' in database definition");
 						successful = false;
 						continue;
 					}
 					
-					final String name = n.getKey();
+					String name = section.getKey();
 					assert name != null;
-					final VariablesStorage s;
-					if (type.equalsIgnoreCase("csv") || type.equalsIgnoreCase("file") || type.equalsIgnoreCase("flatfile")) {
-						s = new FlatFileStorage(name);
-					} else if (type.equalsIgnoreCase("mysql")) {
-						s = new DatabaseStorage(name, Type.MYSQL);
-					} else if (type.equalsIgnoreCase("sqlite")) {
-						s = new DatabaseStorage(name, Type.SQLITE);
-					} else {
+					VariablesStorage storage;
+					Optional<?> optional = types.entries().stream()
```

Comment:
If we end up using a regular map or something, can't we just do a simple Map.get(type)